engine/e2e-test/test_cli_engine_install.py

```diff
@@ -8,7 +8,7 @@ class TestCliEngineInstall:
 
     def test_engines_install_llamacpp_should_be_successfully(self):
         exit_code, output, error = run(
-            "Install Engine", ["engines", "install", "cortex.llamacpp"], timeout=60
+            "Install Engine", ["engines", "install", "cortex.llamacpp"], timeout=None
```

Comment:
Woah - this seems a bit dangerous, I would prefer to have a timeout bound (you can set it at 10x magnitude)

Reply:
I'll update it